src/wrappers/PhylogenyWrapper.js

```diff
@@ -408,11 +408,11 @@ class PhylogenyWrapper {
     const phylogenyAsJSONLD = JSON.parse(JSON.stringify(this.phylogeny));
 
     // Set name and class for phylogeny.
-    phylogenyAsJSONLD['@id'] = baseURI;
+    if (!has(phylogenyAsJSONLD, '@id')) phylogenyAsJSONLD['@id'] = baseURI;
```

Comment:
My only concern here is that this is behavior that should be documented. It seems that currently there is no documentation for ```PhylogenyWrapper.asJSONLD()```. Perhaps because what it does was deemed as obvious from the name, or because the method is considered internal (which would not be evident I think). Either way, it seems to me that the method should get documentation which would include how the parameter will be used depending on circumstances.

Reply:
Good point! Fixed in 871790065c9.